File: uri 1043.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    float A,B,C;
    scanf("%f %f %f",&A,&B,&C);
    if((A+B)>C && (B+C)>A && (C+A)>B)
        printf("Perimetro = %.1f\n",A+B+C);
    else
        printf("Area = %.1f\n",(C*(A+B))/2);
    return 0;
}

File: uri 1043.py
A, B, C = map(float,input().split(" "))
if A+B > C and B+C >A and C+A > B:
    print("Perimetro = %.1f"%(A+B+C))
else:
    print("Area = %.1f"%((C * (A+B)) / 2))
